pycheribuild/projects/cross/busybox.py

```diff
@@ -181,11 +181,23 @@ def setup(self) -> None:
         self.make_args.set(CONFIG_PREFIX=self.install_dir / "rootfs")
         self.make_args.set(MUSL_HOME=self.install_dir)
         self.make_args.set(KHEADERS=self.install_dir / "usr/include/")
-        self.make_args.set(CLANG_RESOURCE_DIR=self.install_dir / "include/lib")
-        # https://git.morello-project.org/morello/morello-busybox/-/merge_requests/1
-        self.make_args.set(CFLAGS="-Wno-error=int-conversion")  # for editors/vi
+        self.make_args.set(CLANG_RESOURCE_DIR=self.install_dir / "lib")
```

Comment:
I wonder if we can just call super().setup() now? And add a few small overrides? But not necessarily for this PR